cmd/node/config/ratings.toml

```diff
@@ -28,19 +28,30 @@
         { MaxThreshold = 10000000,ChancePercent = 24},
     ]
 
-[ShardChain.RatingSteps]
-    HoursToMaxRatingFromStartRating = 72
-    ProposerValidatorImportance = 1.0
-    ProposerDecreaseFactor = -4.0
-    ValidatorDecreaseFactor = -4.0
-    ConsecutiveMissedBlocksPenalty = 1.50
+[ShardChain]
+    [[ShardChain.RatingStepsByEpoch]]
+        HoursToMaxRatingFromStartRating = 72
+        ProposerValidatorImportance = 1.0
+        ProposerDecreaseFactor = -4.0
+        ValidatorDecreaseFactor = -4.0
+        ConsecutiveMissedBlocksPenalty = 1.50
+        EnableEpoch = 0
```

Comment:
Nitpicking - `EnableEpoch` can be the first field.

Reply:
agree, updated